docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+
+     (backported from commit <sha1> <upstream repo name>)
+
+   Please note that the opening and closing parenthesis are required. There
+   must be a brief explanation immediately after the "(backported from ...)"
+   block, between square brackets, with the name of the person who introduced
+   the change::
+
+     (backported from commit <sha1> <upstream repo name>)
+     [roxanan: Had to adjust the context due to missing commit <sha1>]
+
+   If the patch is a simple cherry-pick from an upstream repo and it applies
+   cleanly, that **must** also be spelled out in the provenance section in the
+   form by using 'cherry-picked from'::
+
+     (cherry picked from commit <sha1> <upstream repo name>)
+
+   The opening and closing parenthesis are required here as well.
+
+   .. note::
+
+      "<upstream repo name>" should be omitted if the patch comes from the
+      mainline tree.
+
+   Example::
+
+     Signed-off-by: Adam Jackson <ajax@redhat.com>
+     Signed-off-by: Eric Anholt <eric@anholt.net>
+     Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
+     (cherry picked from commit d4e0018e3e4dd685af25d300fd26a0d5a984482e 2.6.34.y)
+     Signed-off-by: Manoj Iyer <manoj.iyer@canonical.com>
+     Acked-by: Tim Gardner <tim.gardner@canonical.com>
+     Acked-by: Brad Figg <brad.figg@canonical.com>
+     Acked-by: Steve Conklin <sconklin@canonical.com>
+
+#. Every **CVE** patch **must** contain a line at the beginning of the commit
+   message that specifies the CVE number(s) related to the patch. This must be
+   the first part of the body of the comment. There is the comment subject
+   line, a blank line, the CVE number, a blank line and then the rest of the
+   comment body.  A "BugLink" is optional for CVE patches.
+
+   Example::
+
+     Subject: [SRU B/D] UBUNTU: SAUCE: nbd_genl_status: null check for nla_nest_start
+
+     From: Navid Emamdoost <navid.emamdoost@gmail.com>
+
+     CVE-2019-16089
+
+     nla_nest_start may fail and return NULL. The check is inserted, and
+     errno is selected based on other call sites within the same source code.
+     Update: removed extra new line.
+     v3 Update: added release reply, thanks to Michal Kubecek for pointing
+     out.
+     ...
+
+Patch Series
+------------
+
+#. Every patch submitted to a stable kernel **must** be sent in a patch series
+   with a cover letter, even if the patch series contains a single patch.
+
+#. The cover letter **must** contain the "BugLink" or the CVE number like the
+   patch(es) itself.
+
+#. The cover letter **must** contain the SRU justification from the launchpad
+   bug or the CVE fix. See `KernelTeam/KernelUpdates`_.
+
+#. All the emails in the patch series **must** be numbered (e.g. "[PATCH 0/3]",
+   "[PATCH 1/3]", etc.) and all the patches sent in reply to the cover letter
+   (PATCH 0/N). See the option "--no-chain-reply-to" from git-send-email(1).
+
+.. _KernelTeam/KernelUpdates: https://wiki.ubuntu.com/KernelTeam/KernelUpdates
+
+Pull Request
```

Comment:
I think this and the previous section should tell a more cohesive story since they are related to "multiple patches".

Reply:
Reorganized. Thanks!